app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertAdminDashboardDisplayIsProperActionGroup.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminDashboardDisplayIsProperActionGroup">
```

Comment:
I would suggest avoiding something like "proper", "correct" in the naming. Using these keywords might confuse on an attempt to understand what is "proper". I would recommend something like `AssertAdminDashboardDisplayedWithNoErrorsActionGroup`. 

Thanks

Reply:
Has been renamed.